examples/angular-cli/.storybook/preview.ts

```diff
@@ -1,9 +1,10 @@
 import { setCompodocJson } from '@storybook/addon-docs/angular';
 import addCssWarning from '../src/cssWarning';
 
+/* eslint-disable import/extensions, import/no-unresolved */
 // @ts-ignore
-// eslint-disable-next-line import/extensions, import/no-unresolved
 import docJson from '../documentation.json';
+/* eslint-enable import/extensions, import/no-unresolved */
```

Comment:
@shilman What is wrong with this code or with me? :)

Git commit hook fails localy. There is no `documentation.json` in repo. Am I forgot to bootstrap something?

Reply:
`documentation.json` gets generated dynamically when you run storybook in that example